earthpy/tests/test_clip.py

```diff
@@ -100,19 +50,19 @@ def test_clip_poly():
 # TODO same test for points and lines
```

Comment:
oopsy. i believe i did this TODO??

Reply:
Yep - I think that was one of yours in this commit: https://github.com/earthlab/earthpy/commit/77e2ecd28509d138f50a20e51d0dffc2e278fc6d#diff-6c8697d6bac09c332f11fca7ec8cf5ffR87